pkg/cmd/roachtest/spec/cluster_spec.go

```diff
@@ -128,13 +130,7 @@ func getGCEOpts(
 		opts.Zones = zones
 	}
 	opts.SSDCount = localSSDCount
-	if localSSD && localSSDCount > 0 {
```

Comment:
I think this change makes it simpler to understand - multiple stores / single store no matter whether the disks are SSDs or PDs. The same applies to AWS.